### DISCUSSION TITLE
fpm-mode?

### DISCUSSION THREAD

**Discussion by tobycroft** [pj8912/php-bard-api#13] [Announcements]
Is it possible that the project can be run under FPM environment?

**Comment by pj8912** [comment]
not sure, never tried, just basic script to play around

**Comment by tobycroft** [comment]
I am try to make a API base on this , FPM is the most easy way to deploy, or I have to use cli mode maybe with swoole, that would be a little complicated. 

I am still figure a way to do it.

for now, If I deploy this with FPM then the return would be blank.

And I found some issues like this

![image](https://github.com/pj8912/php-bard-api/assets/24893075/97545ea3-219d-41f7-a748-7cf75d1e9535)

Still figuring this

BTW the bard has been upgraded since..... july 23 added the "previous chat", so will u have plan to add the selecting conversation function instead of creating a new one?

**Reply by pixkk** [reply]
This problem occurs when the "SNlM0e" code cannot be found.
